unstructured-pipeline/.env.example

```
@@ -6,8 +6,8 @@ COUCHDB_URL=http://localhost:5984/cms-evidence
 
 # Processing Services
 TIKA_URL=http://localhost:9998
-SOLR_URL=http://localhost:8983/solr/biar
-NIFI_URL=http://localhost:8080/nifi-api
+SOLR_URL=http://localhost:8983/biar_docs
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SOLR_URL key should go before the TIKA_URL key

(UnorderedKey)